src/app/App.js

```diff
@@ -1,20 +1,50 @@
 import React, { Component } from "react";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 
+import Amplify from 'aws-amplify';
+import { withAuthenticator } from 'aws-amplify-react';
+import aws_exports from './aws-exports';
+
 import Map from "./map/Map";
-import Login from "./login/Login";
 import Form from "./form/Form";
 
 import "./App.css";
 
+Amplify.configure(aws_exports);
+
 class App extends Component {
+
+  componentWillMount(){
+    Amplify.Auth.currentAuthenticatedUser().then((user) => {
+      console.log('user',user);
```

Comment:
Do we want to keep this `console.log` in here?

Reply:
I will remove.